package.json

```diff
@@ -2,7 +2,7 @@
     "name": "language-julia",
     "displayName": "Julia",
     "description": "Julia Language Support",
-    "version": "1.0.9",
+    "version": "1.0.10",
```

Comment:
Can you undo this? My tagging workflow automatically bumps version numbers.

Reply:
Done.